client/src/app/site/motions/components/motion-list/motion-list.component.ts

```diff
@@ -41,12 +41,12 @@ export class MotionListComponent extends ListViewBaseComponent<ViewMotion, Motio
     /**
      * Columns to display in table when desktop view is available
      */
-    public displayedColumnsDesktop: string[] = ['identifier', 'title', 'state'];
+    public displayedColumnsDesktop: string[] = ['identifier', 'title', 'anchor', 'state'];
```

Comment:
Why you put 'anchor' not as last column - as in other places (e.g. users)?

Reply:
Oh, my mistake. I first thought, that this will resolve the order of items to render. Put it as last column, now.